Update in-container-configuration.md (#2770)
drud/ddev: docs/users/extend/in-container-configuration.md

## In-Container home directory and shell configuration

> Custom shell configuration (bash or your preferred shell), your usual git configuration, a composer auth.json and more can be achieved within your containers.  Place all your dotfiles in your global`~/.ddev/homeadditions` or your project's `.ddev/homeadditions` directory and DDEV will use these in your project's web containers.  (Note that there's also a hidden homeadditions in the project `.ddev/.homeadditions` where the global `~/.ddev/homeadditions` get copied to, ignore and don't touch this hidden one.)

Reviewer: ```suggestion
Custom shell configuration (bash or your preferred shell), your usual git configuration, a composer auth.json and more can be achieved within your containers.  Place all your dotfiles in your global`~/.ddev/homeadditions` or your project's `.ddev/homeadditions` directory and DDEV will use these in your project's web containers.  (Note that there is also a hidden/transient `.ddev/.homeadditions`; this is used for processing global homeadditions and should be ignored.)
```
